label_sync/labels.yaml

```diff
@@ -353,50 +353,56 @@ repos:
         target: both
         addedBy: label
   tektoncd/dashboard:
+    labels:
       - color: 0052cc
         description: Indicates an issue on api area.
         name: area/api
         target: issues
         addedBy: label
-    - color: 3f6eb5
-      description: Issues related to user-studies
-      name: user-study
-      target: issues
-      addedBy: humans
-    - color: 160a72
-      description: This issue needs some help during design phase
-      name: design-help-wanted
-      target: issues
-      addedBy: humans
+      - color: 3f6eb5
+        description: Issues related to user-studies
+        name: user-study
+        target: issues
+        addedBy: humans
+      - color: 160a72
+        description: This issue needs some help during design phase
+        name: design-help-wanted
+        target: issues
+        addedBy: humans
   tektoncd/pipeline:
+    labels:
+      - color: ffcdc4
+        description: For consideration when planning the next milestone
+        name: maybe-next-milestone
+        target: both
+        addedBy: humans
       - color: 0052cc
         description: Indicates an issue on api area.
         name: area/api
         target: issues
         addedBy: label
-    - color: 3f6eb5
-      description: Issues related to user-studies
-      name: user-study
-      target: issues
-      addedBy: humans
-    - color: 4cdbd6
-      description: This task is mostly about implementation!!! And docs and tests of course but that's a given
-      name: meaty-juicy-coding-work
-      target: both
-      addedBy: humans
-    - color: 3E4B9E
-      description: Issues that should be considered as Epics (aka multiple sub-tasks, …)
-      name: epic
-      previously:
-        - Epic
-      target: both
-      addedBy: humans
-    - color: 64e5ba
-      description: This is for some internal Google project tracking
-      name: okr
-      target: both
-      addedBy: humans
+      - color: 3f6eb5
+        description: Issues related to user-studies
+        name: user-study
+        target: issues
+        addedBy: humans
+      - color: 4cdbd6
+        description: This task is mostly about implementation!!! And docs and tests of course but that's a given
+        name: meaty-juicy-coding-work
+        target: both
+        addedBy: humans
+      - color: 3E4B9E
+        description: Issues that should be considered as Epics (aka multiple sub-tasks, …)
+        name: Epic
+        target: both
+        addedBy: humans
+      - color: 64e5ba
+        description: This is for some internal Google project tracking
+        name: okr
+        target: both
+        addedBy: humans
```

Comment:
i wish we could have a list we share between all the projects without defining it so many times

Reply:
There is one (the default). If we move this one in there, all the repository will have it. I wasn’t sure about keeping it only for pipeline or for all

Reply:
ah kk! didnt know we could do that :D